library/src/main/kotlin/com/chuckerteam/chucker/internal/data/entity/HttpTransaction.kt

```diff
@@ -134,10 +136,13 @@ internal class HttpTransaction(
             return when (status) {
                 Status.Failed -> " ! ! !  $method $path"
                 Status.Requested -> " . . .  $method $path"
-                else -> "$responseCode $method $path"
+                else -> "$responseCode $method $path ${isGraphqlRequest.takeIf { true } ?: ""}"
             }
         }
 
+    val isGraphqlRequest: Boolean
+        get() = !gqlOperationName.isNullOrEmpty()
```

Comment:
This doesn't seem right. The `operationName` is an optional parameter in [GraphQL](https://graphql.org/learn/serving-over-http/#post-request). I think we can rely solely on `query`. Plus there is an additional issue for GET  requests, which I mentioned here.